### DISCUSSION TITLE
How to track parking on public rights of way?

### DISCUSSION THREAD

**Discussion by jfh01** [openmobilityfoundation/mobility-data-specification#656] [Car sharing]
Understanding parking on public streets is one of the key use cases for MDS in car sharing (#640). For example, cities may want to charge fees based on the time that car share vehicles spend in metered spaces. However, determining where a vehicle is parked can be tricky. GPS inaccuracies could lead to off-street parking being categorized as on-street, or vice-versa.

How should MDS approach the challenge of tracking parking on public streets? What data model best represents the various types of parking that are important to capture? How should we address issues of precision and accuracy in data?

**Comment by mbykin** [comment]
I work with parking reporting for Gig Car Share and the data points that we use most are:
- vehicle id - vin, plate, or other unique id
- lat/lng - location
- stop & start timestamps
- trip id @ start and stop - trip id's can be the same at stop & start if member is retaining the car after the stop, or the can be different if one trip is ending and another is starting after the stop
- on/off street - cars can be in public parking locations like meters or residential streets, or they can be in private spaces like lots, depots, or in the shop (for weeks/months); we need the ability to indicate when we know the car is off street
